zerver/openapi/zulip.yaml

```diff
@@ -9089,7 +9089,7 @@ paths:
         - name: message_content_in_email_notifications
           in: query
           description: |
-            Include the message's content in missed messages email notifications.
+            Include the message's content in message email notifications.
```

Comment:
"in email notifications for new messages"?

Reply:
Oh, the "message notification emails" you have in the next block works; just change this to that.